### PR TITLE
[WIP][15.0][MIG] purchase_rfq_number: Migration to V15

### PR DESCRIPTION
**Pull request by bizzappdev**

### CODE REVIEW

File: purchase_rfq_number/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Copyright 2021 ProThai Technology Co.,Ltd. (http://prothaitechnology.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Purchase For Quotation Numeration",
+    "summary": "Different sequence for purchase for quotations",
+    "version": "15.0.0.1.0",
```
Comment (Saran440) [inline purchase_rfq_number/__manifest__.py]:
```suggestion
    "version": "15.0.1.0.0",
```

**Review by Saran440 — COMMENTED** [review]
Functional Test :+1: 

Small fix.

**Review by Saran440 — APPROVED** [review]

**Review by dariodelzozzo — APPROVED** [review]
Code Review

**Pull request merged**